docs/src/content/docs/introduction/overview.mdx

```diff
@@ -67,3 +67,23 @@ When an ASE adopts the Open Payments standard, clients (applications and other p
 - ASEs and clients to create and deploy custom integrations to communicate with one another
 
 Open Payments aims to simplify and democratize payments by providing a standardized, easy-to-integrate solution. This reduces development effort and enhances financial inclusion by making payment solutions more affordable and accessible to all.
+
+## Use cases
+
+Open Payments facilitates various use cases including but not limited to:
+
+### Peer-to-peer payment transfers
+
+As mentioned above, a sender with an account at one Open Payments-enabled ASE can send a payment to a recipient with an account at a different Open Payments-enabled ASE by sending the payment from the sender’s wallet address to the recipient’s wallet address. Thus, Open Payments brings a similar user experience to payments as email brings to person-to-person communications.
+
+### E-commerce platforms
+
+Open Payments can facilitate one-time payments directly between merchants and customers or via third-party marketplaces. A key difference between conventional e-commerce platforms and those enabled by Open Payments is that the payment function won’t be in the control of a sole payment processor or gateway. Refer to the [one-time payment guide](/guides/make-onetime-payment) for more details.
```

Comment:
Is this use case for e-commerce platforms or for one-time payments? Could an e-commerce platform support both one-time and recurring payments?

I suggest working with a dev familiar with Open Payments -- doesn't have to be Max -- and see if together you can come up with a more robust use case. I do like that you mention how OP-enabled platforms don't necessarily need to rely on a single payment processor/gateway.

Reply:
Also, side thought, is the one-time payment guide applicable to peer-to-peer?

Reply:
Leveraged text from use case on Rafiki docs and added links to one-time, recurring guides and Max's e-commerce example.  Also added one-time payment guide to peer-to-peer